src/scripts.js

```diff
@@ -11,28 +11,65 @@ import User from './user';
 import Cookbook from './cookbook';
 
 let favButton = document.querySelector('.view-favorites');
-let homeButton = document.querySelector('.home')
+let recipeToCookButton = document.querySelector('.view-recipe-to-cook');
```

Comment:
Is this referencing one recipe to cook or the list of recipes to cook? If it's the list, the naming should be plural again - recipesToCookButton, viewRecipesToCook(), searchRecipes(), etc. Not a big deal, just a little naming convention improvement we could make :)

Reply:
I will make the changes in those naming conventions.  Good eye Christina!